### ISSUE TITLE
Is it possible to define variables in the ini configuration files to be re-used by the tracks

### ISSUE THREAD

**Issue by rikrdo89** [deeptools/pyGenomeTracks#316]

**Comment by rikrdo89** [comment]
Sorry this is more a enhancement suggestion that an issue, but I was wondering if there is a way to define some global variables to avoid a of typing.. I envision something like this

[myGlobalVars]
myMin = 2
myMax = 10
myColor = Blue

[a sample track]
color = %(myColor)
min_value = %(myMin)

**Comment by lldelisle** [comment]
Hi,
Thanks for the idea, I think this would be like #219 . We would like to implement it but for the moment we are lacking time to develop it...

**Comment by lldelisle** [comment]
Sorry, rereading it more carefully this is another idea. I prefer the idea of changing the default than defining variables. Could you tell me if #219 would be ok for you of if you have some cases of variables would help?

**Comment by rikrdo89** [comment]
I think #219 would be helpful as well... honestly anything that would allow the user to automate some of the configuration changes instead of going line by line in the configuration file would be super useful.

**Comment by lldelisle** [comment]
@rikrdo89 I agree, for the moment, I don't have time to work on pyGenomeTracks, unfortunately.
